### PR TITLE
LEAN-2585: AR-020 - Restrict Author details editing - FE implementation

### PR DESCRIPTION
**Pull request by nghazzawi-atypon**

### CODE REVIEW

File: src/lib/capabilities.tsx [hunk 1]
```diff
@@ -142,7 +144,7 @@ export const getCapabilities = (
     formatArticle: !isViewer(),
     /* editor */
     editArticle: !isViewer(),
-    editMetadata: !(isViewer() || isAnnotator()),
+    editMetadata: !(isViewer() || isProofer()),
```
Comment (asouqi) [inline src/lib/capabilities.tsx:147]: why remove Annotater, as he can't change any model. so now the backend will allow it!

Comment (asouqi) [inline src/lib/capabilities.tsx:147]: why remove Annotater, as he can't change any model. so now the backend will allow it!

**Review by asouqi — CHANGES_REQUESTED** [review]
please check editMetadata

File: src/lib/capabilities.tsx (hunk 1 above)
Comment (nghazzawi-atypon) [reply src/lib/capabilities.tsx:147]: @asouqi this is was requested on https://jira.atypon.com/browse/LEAN-2584

-The Annotator role should have access to the "edit authors" button

The Annotator role should be able to click on the Authors' links to edit the details

**Comment by nghazzawi-atypon** [comment]
@asouqi Based on my understanding, it seems that we are using it to control the enabling or disabling of an "edit author" button and author links. The functionality is determined by the value of the can.editMetadata variable.

There was a request made to grant the Annotator access to all of these features. The request can be found on the following Jira ticket: https://jira.atypon.com/browse/LEAN-2584.

File: src/lib/capabilities.tsx (hunk 1 above)
Comment (asouqi) [reply src/lib/capabilities.tsx:147]: if they didn't change that in the backend that will not work, annotator can't change MPModel

**Review by asouqi — APPROVED** [review]

**Pull request merged**